addons/website_slides/static/src/scss/slides_slide_fullscreen.scss

```diff
@@ -92,17 +92,27 @@
 
             .o_wslides_sidebar_content {
                 line-height: unset;
+
+                .list-unstyled i {
+                    color: $gray-600
+                }
+
                 i {
                     line-height: unset;
+
+                    &.fa-flag-checkered {
+                        padding-left: 3px;
+                        padding-right: 7px;
+                    }
                 }
             }
 
             .o_wslides_sidebar_done_button {
                 line-height: unset;
 
                 button {
-                    padding-top: 0;
-                    border-top: 0;
+                    padding: 0;
+                    padding-right: 3px;
```

Comment:
Can't this padding be replaced by bootstrap classes, as it does not seem "complex" ?

Reply:
Thanks for looking into it. This design is a bit old, so when it will be updated some inconsistencies will be removed. For now we made the minimal diff to solve current UX issues, so ok :) .